pimd/pim_cmd.c

```diff
@@ -3119,18 +3119,13 @@ static void pim_show_group_rp_mappings_info(struct pim_instance *pim,
 	json_object *json_group = NULL;
 	json_object *json_row = NULL;
 
-	if (pim->global_scope.current_bsr.s_addr == INADDR_ANY)
-		strlcpy(bsr_str, "0.0.0.0", sizeof(bsr_str));
-
-	else
-		pim_inet4_dump("<bsr?>", pim->global_scope.current_bsr, bsr_str,
-			       sizeof(bsr_str));
-
 	if (uj) {
 		json = json_object_new_object();
-		json_object_string_add(json, "BSR Address", bsr_str);
+		json_object_string_add(json, "BSR Address", "%pPAs",
```

Comment:
It's `json_object_string_addf`, extra `f` for format support

Reply:
Taken care